tutorials/hana-cloud-alerts/hana-cloud-alerts.md

```diff
@@ -114,7 +114,7 @@ The following instructions demonstrate a few examples of triggering alerts in an
     END;
     ```
 
-    > Long running statements can be run as a background activity which will continue even if the machine the SAP HANA database explorer is running on loses internet connectivity or the tab is closed.  
+    >Long running statements can be run as a background activities in the SAP database explorer which will continue even if the browser's connection drops or the tab is closed.  
```

Comment:
"run as a background activities in the SAP database"
Please remove "a" or change "activities" to singular.

Reply:
Fixed